setup.py

```diff
@@ -75,12 +75,15 @@ def get_version(*file_paths):
         'Development Status :: 5 - Production/Stable',
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.2',
+        'Framework :: Django :: 4.2',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: GNU Affero General Public License v3',
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
```

Comment:
Same with python 3.5, we should remove it

Reply:
Done

Reply:
I think you forgot to push the commit that removes these lines.